### PR TITLE
chore: create assessment and bug issue template

### PR DESCRIPTION
**Pull request by pbiology**
### This PR:

Added: 
- An issue template for assessments
- An issue template for bugs

### CODE REVIEW

**Review by mathiasbio — APPROVED** [review]
Nice! Looks good to me! 🌟

File: .github/ISSUE_TEMPLATE/assessment.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Assessment
+description: An assessment for BALSAMIC
+title: "[Assessment] <title>"
+labels: ["Assessment"]
+projects: ["Clinical-Genomics/46"]
```
Comment (ivadym) [inline .github/ISSUE_TEMPLATE/assessment.yml:5]: Is this field to link the project-planning issues?

File: .github/ISSUE_TEMPLATE/assessment.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- type: textarea
+  attributes:
+    label: Criteria
+    description: List the criteria which will be evaluated
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Origin
+    description: Link to tickets, issues, discussion etc from where this originated
```
Comment (ivadym) [inline .github/ISSUE_TEMPLATE/assessment.yml:28]:
```suggestion
    description: Link to tickets, issues, discussion, etc., from where this originated.
```

**Review by ivadym — APPROVED** [review]
Looks good🔥

File: .github/ISSUE_TEMPLATE/assessment.yml (hunk 2 above)
Comment (ivadym) [reply .github/ISSUE_TEMPLATE/assessment.yml:28]: I'm too slow as a reviewer 🐌

**Pull request merged**